bin/console

```diff
@@ -2,5 +2,6 @@
 
 require File.expand_path('../config/boot', __dir__)
 ENV['RAILS_ENV'] ||= 'production'
+ENV['DB_CONNECTION_STRING'] ||= 'mysql2://root:password@localhost:3306/cc_test_'
```

Comment:
Couple of issues here:
1. When I tried this, I needed to set the DB to `cc_test` instead of `cc_test_`
2. Your PR doesn't check to see if `MYSQL_CONNECTION_PREFIX` or `POSTGRES_CONNECTION_PREFIX` are set before assuming the standard DB_CONNECTION_PREFIX (see https://github.com/cloudfoundry/cloud_controller_ng/pull/1117 )

Reply:
Here's what I would do:
```
#!/usr/bin/env ruby

def ensure_db_connection_string
  ENV['DB_CONNECTION_STRING'] ||=
    if ENV['POSTGRES_CONNECTION_PREFIX']
      ENV['POSTGRES_CONNECTION_PREFIX'] + '/cc_test'
    elsif ENV['MYSQL_CONNECTION_PREFIX']
      ENV['MYSQL_CONNECTION_PREFIX'] + '/cc_test'
    else
      'mysql2://root:password@localhost:3306/cc_test'
    end
end

require File.expand_path('../config/boot', __dir__)
ENV['RAILS_ENV'] ||= 'production'
ensure_db_connection_string

require 'cloud_controller/console'
```

Don't know what's going on with that prefix. Does this change work for you?

@ericpromislow